### PR TITLE
Update

### PR DESCRIPTION
**Pull request by heyhimansh**

### CODE REVIEW

**Comment by heyhimansh** [comment]
task 1

**Comment by techsavvyash** [comment]
what is the intent of this PR?

File: schema.prisma
Comment (techsavvyash) [inline schema.prisma:1]: why do we have a `schema.prisma` here?

File: query.js
Comment (techsavvyash) [inline query.js:1]: what's the point of this file?

File: .github/workflows/classroom.yml [hunk 1]
```diff
@@ -1,19 +1,220 @@
...
+        setup-command: npm install
+        command: npm run test:1
+        timeout: 10
+        max-score: 10
+    - name: Step-2 Test
+      id: step-2-test
+      uses: education/autograding-command-grader@v1
+      with:
+        test-name: Step-2 Test
+        setup-command: npm install
```
Comment (techsavvyash) [inline .github/workflows/classroom.yml:31]: why do we need to install the dependencies before each test? 🤔

File: .github/workflows/classroom.yml
Comment (techsavvyash) [inline .github/workflows/classroom.yml:1]: what do you think id the problem in the current github workflows?

File: .github/.keep
Comment (techsavvyash) [inline .github/.keep:1]: why do we need this if we already have a `workflows/classroom.yml` in the `.github/` folder?

**Review by techsavvyash — CHANGES_REQUESTED** [review]

File: .github/workflows/classroom.yml
Comment (nithieshnaik) [reply .github/workflows/classroom.yml:1]: i did not get you

Comment (nithieshnaik) [reply .github/workflows/classroom.yml:1]: i did not get you

Comment (nithieshnaik) [reply .github/workflows/classroom.yml:1]: i recently applied for github class so i did not get the id

File: .github/.keep
Comment (sidpg123) [reply .github/.keep:1]: in git, empty directories are not tracked, as git looks for changes in folders not in files. But if we want to add an empty directory in our repository we can create .keep file.

Comment (techsavvyash) [reply .github/.keep:1]: the `.github` folder already has a workflows folder we don't need the `.gitkeep` file its just to keep empty folders.

**Comment by techsavvyash** [comment]
@heyhimansh you don't have to raise a PR for each step, just keep committing and pushing to the main branch of your fork and your scores will be calculated. Closing this PR.

**Pull request closed without merging**